l10n_it_account/tests/test_l10n_it_account.py

```diff
@@ -128,3 +128,11 @@ def test_vat_22_50(self):
         self.assertEqual(tax.balance, -22)
         self.assertEqual(tax.deductible_balance, -11)
         self.assertEqual(tax.undeductible_balance, -11)
+
+    def test_format_vat_it(self):
+        partner_id = self.env.ref("base.res_partner_12")
```

Comment:
Pare che questo test passi anche senza il metodo che hai aggiunto a `res.partner`, puoi verificare?

Se aggiungi un test per una modifica, il test dovrebbe fallire senza la tua modifica e passare con essa.

Reply:
Ero abbastanza sicuro che fallisse senza, anche perché mi ero scordato di mettere from . import res_partner in models/__init__.py e ho litigato 5 minuti col debugger prima di capire perché la funzione venisse ignorata. E il test falliva. Non mi pare di aver modificato altro dopo.   Domani controllo.

Reply:
Stamattina in chiamata abbiamo verificato che questo test passa anche senza il nuovo metodo che hai aggiunto a `res.partner`, vedi tu se chiudere tutto o aggiungere solo il test